docs/data/pages.ts

```diff
@@ -532,7 +532,16 @@ const pages: MuiPage[] = [
             title: 'Heatmap',
             plan: 'pro',
           },
-          { pathname: '/x/react-charts/funnel', title: 'Funnel', plan: 'pro', unstable: true },
+          {
+            pathname: '/x/react-charts-funnel',
+            title: 'Funnel',
+            plan: 'pro',
+            unstable: true,
+            children: [
+              { pathname: '/x/react-charts/funnel', title: 'Funnel Overview' },
+              { pathname: '/x/react-charts/pyramid', title: 'Pyramid demo' },
+            ],
+          },
```

Comment:
Does It make sense for it to be a child of funnel, marketing-wise? It could also be a standalone section by itself.

Reply:
We might not have enough content to make it a standalone parent page. So, for now I'll keep it as a child.